docs/rgw_setup.md

```diff
@@ -0,0 +1,136 @@
+# Setting up Ceph Object Gateway for different RPC Use Cases
+
+Ceph Object Gateway(rgw) is an object storage interface built on top of librados to provide applications with a RESTful gateway to Ceph Storage Clusters. Ceph Object Storage supports two interfaces  S3 and OpenStack Swift.   
+
+Complete documentation for the Ceph Object Gateway can be found (here.)[http://docs.ceph.com/docs/master/radosgw/]
+
+## Role of rpc-ceph
+The purpose of (rpc-ceph)[https://github.com/rcbops/rpc-ceph] is to provide a consistant versioned set of processes, variables and versioned software for RPC's use.  
+
+### List of Use Cases
+
+* General Installation using ceph-ansible
+* Configuring rgw to use a Stand alone instance of OpenStack Keystone
+* Configuring rgw to use KeyStone and Interact with an OpenStack environment in place of OpenStack Swift
+
+
+
+## Installation of rgw only
+To Install rgw **ONLY** and **NOT** use any of rpc-ceph configured varibles 
```

Comment:
The first use case has nothing to do with KeyStone or OpenStack.

Reply:
Agreed, but rpc-ceph has vars for ceph in general, this line implies we're not setting any vars at all. Also, since the vars will still work as long as you haven't set "keystone_radosgw: True" the deployer/user will still get the defaults that we set here, without any of the keystone integration.

Reply:
Sounds like you would rather we do away with the first use case all together.    That's cool.

Reply:
I removed the first use case